samples/android/app/src/main/java/org/hyperledger/indy/sdk/helpers/MessageHelper.kt

```diff
@@ -1,21 +1,76 @@
 package org.hyperledger.indy.sdk.helpers
 
+import android.app.Activity
 import android.content.Context
+import android.graphics.Color
+import android.view.View
+import android.widget.ProgressBar
+import android.widget.TextView
 import android.widget.Toast
+import androidx.core.content.ContextCompat
 import es.dmoral.toasty.Toasty
+import kotlinx.android.synthetic.main.activity_anoncreds.*
+import org.hyperledger.indy.sdk.R
 
 public class MessageHelper {
```

Comment:
no need for class here. Make it an object.

Reply:
I would put logic that you have here in parent activity that all other activities extend.